docs/blog/Webots-2021-b-release.md

```diff
@@ -0,0 +1,111 @@
+---
+
+It's that time of year again! Today we're happy to announce the release of Webots R2021b!
+And it's packed with some new features, improvements and, of course, bug fixes.
+Here we are going to present some of the main new features, but for a comprehensive list of changes please refer to the [ChangeLog](../reference/changelog-r2021.md).
+
+
+## Improved Support for Reinforcement Learning
+
+We added a set of new functionalities to make it easier to run reinforcement learning experiments.
+Other than reducing the Webots package size to speed up installation on servers and virtual containers, some additional Supervisor API functions to better manage the reset of the simulation has been included.
+In particular, it is now possible to save the state of a given node and all its descendant nodes and restore it later during the simulation run or export a node definition as string to duplicate the node a later point.
+Then, it is now also possible to artificially and immediately set the position of active and passive joints without having to activate the motor.
+
+An example showing the integration of OpenAI Gym with Webots is now available in our samples library.
+
+---
+
+## Reduce Package Size
+
+For this release we work hard at finding solutions to reduce the size of the Webots installation packages.
+To this end, we removed textures, meshes and sounds from the package. These resources are now downloaded from the web only when needed and cached for subsequent use.
+Additionally, we removed the offline documentation and ..
+
+TODO (update with package size difference).
+
+---
+
+## New Web Rendering Engine
+
+Since the introduction of the web interface for streaming simulations and playing recorded animations, we struggled finding optimal parameters for shadows and illuminations that will reproduce exactly the same rendering as in the Webots Desktop application.
+Finally, we decided to port our custom 3D renderer WREN to WebAssembly and use it for the web simulation interface in order to minimize the differences with the Desktop version and make it straightforward to run high quality simulations and animations on the web.
+
+During this work, we also took the time to refresh and improve the interface.
+
+%figure "New Wen Simulation Interface"
+![Web Simulation Interface](images/web_simulation_interface.png)
+%end
+
+---
+
+## Extended Motor and Joint Functionalities
+
+The [Motor](../reference/motor.md) and [Joint](../reference/joint.md) nodes have been extended to provide more realistic simulations.
+
+We enhanced the `Motor` node to support coupled motors specifying the linked motor in the `name` field and setting their ratio using the new `multiplier` field.
+
+Then, we added two new PROTO nodes [HingeJointWithBacklash](../guide/hinge-joint-with-backlash) and [Hinge2JointWithBacklash](../guide/hinge-2-joint-with-backlash) to easily add backlash effect and a [Gear](..//doc/guide/object-gear.md) model to simulate collision-based transmission.
```

Comment:
```suggestion
Then, we added two new PROTO nodes: [HingeJointWithBacklash](../guide/hinge-joint-with-backlash) and [Hinge2JointWithBacklash](../guide/hinge-2-joint-with-backlash) to easily add backlash effect and a [Gear](..//doc/guide/object-gear.md) model to simulate collision-based transmission.
```

Reply:
I would not add the semi colon here. Otherwise the sentence doesn't make sense anymore given that it also specifies the gear proto.

Reply:
OK, then this is probably better:
```suggestion
Then, we added three new PROTO nodes. The [HingeJointWithBacklash](../guide/hinge-joint-with-backlash) and [Hinge2JointWithBacklash](../guide/hinge-2-joint-with-backlash) allow users to easily add a backlash effect in hinges. The [Gear](..//doc/guide/object-gear.md) model can be used to simulate a collision-based transmission.
```

Reply:
I would rather split the two sentences.
I would change it when I'm back.